server/configuration/example.yaml

```diff
@@ -26,3 +26,13 @@
 #   username: "postgres"
 #   password: "password"
 #   database_name: "omsupply-database"
+# logging:
+##   one of: Console | File
+#   destination: Console
```

Comment:
destination sounds like "file destination" maybe call it logging "mode"?

Reply:
thanks - will update